docs/sources/setup-grafana/configure-grafana/_index.md

```diff
@@ -904,6 +904,21 @@ The following table shows the OAuth provider's setting with the default value an
 | Google | false | true | User organization roles are set with `defaultRole` and the org role can be changed for Google synced users. |
 | Google | true | true | User organization roles are set with `defaultRole` for Google. For other providers, the synchronization will be skipped, and the org role can be changed, along with other OAuth provider users' org roles. |
 
+### [auth.github] skip_org_role_sync
+
+Upon the first login from a user, we set the organization roles from the setting `AutoAssignOrgRole`. If you want to manage organizational roles, set the `skip_org_role_sync` option to `true`. Github syncs organizational roles and sets GrafanaAdmins as well.
```

Comment:
what does "...sets Grafana Admins as well" mean?

Reply:
ah it means the role to be a GrafanaAdmin not just "Admin" as a user.